docs/metrics/accelerate.md

```diff
@@ -63,6 +63,15 @@ average(incident resolved timestamp - incident created timestamp)
 
 Pulse uses the maximum value over a period of time to display aggregate times to recover.
 
+<!--match-incident-deployment-start-->
+!!! note
+    We decided to avoid requiring a relationship between **incidents** and **deployments** or **changes** to simplify the data model reported.
+
+    As such, Pulse considers that the **deployment** for a system immediately before an **incident** for the same system is the one that caused that incident. The same deployment might be responsible for multiple incidents.
```

Comment:
```suggestion
    As such, Pulse considers that the **deployment** for a [system](https://docs.pulse.codacy.com/cli/cli/#before-you-begin) immediately before an **incident** for the same system is the one that caused that incident. The same deployment might be responsible for multiple incidents.
```
I'd consider linking here, unless we're positive the user knows what a system is.
I'm new to these docs and this had me confused, had to dig around.

Reply:
Good suggestion, I applied it.